### PR TITLE
physunits 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by cxwx**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/physunits.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class Physunits < Formula
+  desc "C++ header-only for Physics unit/quantity manipulation and conversion"
+  homepage "https://github.com/martinmoene/PhysUnits-CT-Cpp11"
+  url "https://github.com/martinmoene/PhysUnits-CT-Cpp11/archive/v1.2.0.tar.gz"
+  # version "v1.2.0"
```
Comment (SMillerDev) [inline Formula/physunits.rb:5]: Not needed

Comment (cxwx) [reply Formula/physunits.rb]: changed

**Comment by chenrui333** [comment]
notability audit check failure

**Comment by cxwx** [comment]
* GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)

**Pull request closed without merging**